moto/ec2/models.py

```diff
@@ -9,6 +9,8 @@
 import six
 import warnings
 
+from typing import List, Tuple, Dict, Optional
```

Comment:
Moto still supports Python2, so PEP 484 (type hints) is not supported at all

Reply:
Type hints have been removed